styles/output/anatomy-pdf.css

```diff
@@ -2379,6 +2381,7 @@ h4.os-subtitle + .os-figure:not(.has-splash) > figure {
   text-decoration: none;
   color: #000000;
   font-weight: bold;
+  min-width: 24px;
```

Comment:
Changes with `min-width` are from different PR. Probably styles were not built for all books.